bin/stop

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -e
+docker-compose stop
```

Comment:
🙋‍♂️ We could use `down` instead if the aim is to clean up as `stop` does not remove the containers.

Reply:
:thumbsup: